packages/turf-center-median/test.js

```diff
@@ -0,0 +1,37 @@
+import test from 'tape';
+import glob from 'glob';
+import path from 'path';
+import load from 'load-json-file';
+import write from 'write-json-file';
+import { featureCollection } from '@turf/helpers';
+import centerMean from '@turf/center-mean';
+import centerMedian from '.';
+
+test('turf-center-median', t => {
+    glob.sync(path.join(__dirname, 'test', 'in', '*.json')).forEach(filepath => {
+        // Define params
+        const {name} = path.parse(filepath);
+        const geojson = load.sync(filepath);
+        const options = {
+          weight: geojson.properties.weight,
+          tolerance: geojson.properties.tolerance
+        };
+        let meanCenter = centerMean(geojson, options);
+        meanCenter.properties['marker-color'] = '#a00';
```

Comment:
Try to use a custom helper function `colorize()` which removes any "noise" from this test case, it makes it easier to read this code block.